lib/whitehall_importer.rb

```diff
@@ -82,10 +82,14 @@ def self.sync(whitehall_import)
   end
 
   def self.create_timeline_entry(edition)
-    TimelineEntry.create_for_revision(
+    whitehall_migration_timeline_entry = TimelineEntry::WhitehallImportedEntry.create!(
```

Comment:
minor: we could probably could this something more succinct like `details` given how it's used rather than the long variable name.